src/api/app/models/project.rb

```diff
@@ -968,6 +976,44 @@ def branch_to_repositories_from(project, pkg_to_enable, opts = {})
     end
   end
 
+  def branch_remote_repositories(project)
```

Comment:
but the branch local repositories method was already in this method. So in case that we want to put this method out we should move more things too. Why don't we create a card for the next Sprint to take a look to what we can put outside the Project model instead adding a comment nobody will probably read again?